src/components/Dashboard.jsx

```diff
@@ -141,31 +145,25 @@ class Dashboard extends React.Component {
             statuses={statuses}
           />
           {Object.keys(endpoints).map((endpointName) => {
-            // Return the element. Also pass key
-            const endpoint = endpoints[endpointName];
-            return (
-              <StatusItem
-                key={endpointName}
-                serviceName={endpoint.displayName}
-                serviceStatus={endpoint.status}
-                statusMessage={statuses[endpoint.status].message}
-                statusIcon={statuses[endpoint.status].icon}
-              />
-            );
+             // Return the element. Also pass key
```

Comment:
I'm seeing some extra indention here that may not be needed. Could this be looked into? Probably an editor gone awry  🤖

Reply:
Er how's this @mejackreed?